packages/core/src/types.ts

```diff
@@ -1792,7 +1778,10 @@ export interface ActorRef<TEvent extends EventObject, TSnapshot = any>
   // TODO: figure out how to hide this externally as `sendTo(ctx => ctx.actorRef._parent._parent._parent._parent)` shouldn't be allowed
   _parent?: ActorRef<any, any>;
   status: ActorStatus;
-  src?: InvokeSourceDefinition;
+
+  // Meta data about the actor
+  src?: string;
+  input?: any;
```

Comment:
I think that `input` should be removed from here. Once we pass it to the actor when computing the initial state it should become "protected". It's up to the actor itself to expose it on its snapshot if needed.

I also don't see why `src` should be part of the public information about the actor's instance.